src/skin/skinloader.cpp

```diff
@@ -56,12 +56,21 @@ QString SkinLoader::getSkinPath(const QString& skinName) const {
     return QString();
 }
 
-QPixmap SkinLoader::getSkinPreview(const QString& skinName) const {
-    QPixmap preview(getSkinPath(skinName) + "/preferences_preview_screenshot.png");
+QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schemeName) const {
+	QPixmap preview(":/images/skin_preview_placeholder.png");
+    if (!schemeName.isNull()) {
+    	QString schemeNameUnformatted = schemeName;
+    	QString schemeNameFormatted = schemeNameUnformatted.replace(" ","");
+    	QPixmap preview(getSkinPath(skinName) + "/skin_preview_" + schemeNameFormatted + ".png");
+    	qDebug() << "Looking for: " << getSkinPath(skinName) << "/skin_preview_" << schemeNameFormatted << ".png";
+
```

Comment:
unfortunately I'm poking in the dark here as call hierarchy doesn't work in my eclipse install